config/locales/jobseekers.yml

```diff
@@ -85,9 +85,12 @@ en:
           heading: Personal details
           title: Personal details — Application
         personal_statement:
+          additional_instructions: Additional instructions from the school hiring staff
+          banner_title: Job listing and supporting documents
+          banner_description_html: Documents supplied with `%{job_link}` listing that may be of some help with your personal statement.
```

Comment:
Please use these nice curly quotes: 
Left ‘
Right ’

These are the ones used here:
<img width="483" alt="Screenshot 2021-03-02 at 17 58 39" src="https://user-images.githubusercontent.com/60350599/109692875-ec92b500-7b80-11eb-9e7e-8d43f358a5ee.png">